odoo/addons/base/tests/test_translate.py

```diff
@@ -1105,6 +1105,58 @@ def test_sync_xml_collision(self):
         self.assertEqual(view.with_env(env_fr).arch_db, archf % (terms_fr[0], terms_en[1], terms_en[2]))
         self.assertEqual(view.with_env(env_nl).arch_db, archf % (terms_nl[0], terms_en[1], terms_en[2]))
 
+    def test_sync_xml_inline_modifiers(self):
+        """ Check translations of 'arch' after xml tags changes in source terms
+            when source term had updated modifiers attributes
+        """
+        archf = '''<form class="row">
+    %s
+    <div class="s_table_of_content_vertical_navbar" data-name="Navbar" contenteditable="false">
+        <div class="s_table_of_content_navbar" style="top: 76px;"><a href="#table_of_content_heading_1672668075678_4" class="table_of_content_link">%s</a></div>
+    </div>
+    <div class="s_table_of_content_main" data-name="Content">
+        <section class="pb16">
+            <h1 data-anchor="true" class="o_default_snippet_text" id="table_of_content_heading_1672668075678_4">%s</h1>
+        </section>
+    </div>
+</form>'''
+        terms_en = ('Bread and cheese', 'Knive and Fork', 'Knive <span style="font-weight:bold">and</span> Fork')
```

Comment:
Yes, I wanted to ensure nothing is lost. Also reuse the structure of the other tests since _in principle_ it makes reading one test after the other simpler. In order to simplify the test I may need to change more than only the terms as done here.